src/handlers/channel_item_handler.erl

```diff
@@ -58,7 +58,8 @@ from_json(Req, State = #{is_new := true,
      } when is_list(DaysBefore), length(DaysBefore) =< 5 ->
       CleanConfig = maps:filter(fun not_empty_value/2, Config),
       {ok, _} = db_channel:create(Email, Name, Type, CleanConfig, DaysBefore, Time, TimeZone),
-
+      {ok, Channel} = db_channel:get(Email, Name),
```

Comment:
db_channel:create already returns the channel. You should understand the previous code before introducing a new line.